### ISSUE TITLE
Running locally (for testing) requires setting GIT_REPO_URL

### ISSUE THREAD

**Issue by bruceadams** [rust-lang/crates.io#2030]
Running the `background-worker` locally fails to find the local index git repo, like this:

```bash
$ cargo run --bin background-worker
    Finished dev [unoptimized + debuginfo] target(s) in 0.25s
     Running `target/debug/background-worker`
Booting runner
Using local uploader, crate files will be in the local_uploads directory
Cloning index
thread 'main' panicked at 'Failed to clone index: Error { code: -1, klass: 2, message: "failed to resolve path \'file://./tmp/index-bare\': No such file or directory" }', src/libcore/result.rs:1165:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

A workaround, described in https://github.com/rust-lang/crates.io/pull/1996#issuecomment-567920787, is to define GIT_REPO_URL to the correct, absolute path on the local machine, like this:

```bash
$ GIT_REPO_URL=file:///Users/bruce/github/crates.io/tmp/index-bare cargo run --bin background-worker
    Finished dev [unoptimized + debuginfo] target(s) in 0.27s
     Running `target/debug/background-worker`
Booting runner
Using local uploader, crate files will be in the local_uploads directory
Cloning index
Runner booted, running jobs
```

Ideally, this would manage to _just work_ using relative paths.

**Comment by carols10cents** [comment]
Closing because #2051 was merged; looks like the auto-close stuff didn't work for some reason?

**Comment by smarnach** [comment]
The PR stated

> This fixes issue #2030

GitHub only looks at the word immediately before the bug number to see whether it's [one of the trigger words](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords#about-issue-references), but "issue" isn't one of them.

**Comment by bruceadams** [comment]
Oops! I guess I don't speak Github very well. 🙊
